lib/puppet/face/help.rb

```diff
@@ -146,7 +146,9 @@ def all_application_summaries()
       if (is_face_app?(appname))
         begin
           face = Puppet::Face[appname, :current]
-          result << [appname, face.summary]
+          # Add deprecation message to summary if the face is deprecated
+          summary = face.respond_to?(:deprecated?) && face.deprecated? ? face.summary + _(" (Deprecated)") : face.summary
```

Comment:
I was concerned about the comment "both legacy and Face applications" so I was being a little cautious about the evaluation.

Can definitely drop it though.